ext/curl/tests/curl_multi_info_read.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+array curl_multi_info_read ( resource $mh [, int &$msgs_in_queue = NULL ] );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (!extension_loaded('curl'))
+    print("skip");
+?>
+--FILE--
+<?php
+$urls = array(
+    "it.yahoo.com/",
+    "fr.yahoo.com/"
```

Comment:
Could you use our domains (php.net), in these tests?

docs.php.net
bugs.php.net
...